src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState, useEffect } from "react"
+import { getArticleById } from "../api"
+
+const ArticleCard = (props) => {
+    const { article } = props
+    const [articleCard, setArticleCard] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
+    const [isError, setIsError] = useState(false)
+    
+    useEffect(() => {
+        setIsLoading(true)
+        getArticleById(article.article_id)
+        .then((articleCard) => {
+            setArticleCard(articleCard)
+            setIsLoading(false)
+        })
+        .catch((err) => {
+            setIsError(true)
+        })
+    },[])
+
+    if (isLoading) {
+        return <p>Loading...</p>
+    }
+    
+    if (isError) {
+        return <p>Sorry, article not found</p>
+    }
+
+    const shortDate = article.created_at.slice(8,10)+"/"+article.created_at.slice(5,7)+"/"+article.created_at.slice(0,4)
+    const formattedDate = `${shortDate} ${article.created_at.slice(12,16)}`
+    
+    return (
+        <section id="article-card">
+            <ul className="display-line1">
```

Comment:
I'm not sure UL is descriptive of what these tags contain. Think about if this was being read by a screenreader to a non-sighted person. They would think each article was a series of lists of things 😱